langfuse/test/langfuse-openai.spec.ts

```diff
@@ -0,0 +1,361 @@
+import OpenAI from "openai";
+import { OpenAIWrapper } from "../index";
+import { randomUUID } from "crypto";
+import axios, { AxiosResponse } from "axios";
+import { LANGFUSE_BASEURL, getHeaders } from "../../integration-test/integration-utils";
+import Langfuse from "../index";
+
+const openai = new OpenAI();
+
+const getGeneration = async (name: string): Promise<AxiosResponse<any, any>> => {
+    const url = `${LANGFUSE_BASEURL}/api/public/observations?name=${name}&type=GENERATION`
+    const res = await axios.get(url, {
+        headers: getHeaders(),
+    });
+    return res
+}
+
+describe("Langfuse-OpenAI-Intergation", () => {
```

Comment:
These additional tests would be helpful
- trace that's created covering i/o and other attributes that can be set in the wrapper
- test where openai is wrapped once but used multiple times

Reply:
Updated the test cases and added more validations along with newer tests.